machines/openIdBle/scan.ts

```diff
@@ -150,23 +155,15 @@ export const scanMachine =
           states: {
             checking: {
               invoke: {
-                src: 'checkBluetoothService',
+                src: 'checkBluetoothPermission',
```

Comment:
I see we added permission check only to scan flow. Any idea why it is not implements in request flow also?